examples/poolid-filter/README.md

```
@@ -0,0 +1,83 @@
+Or with custom environment variables:
+
+```bash
+export CARDANO_NODE_SOCKET_PATH=/path/to/node.socket
+export CARDANO_NODE_MAGIC=764824073
+go run main.go
+```
+
+## Expected Output
+
+The program will only output events related to the specified pools:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)